docs/getting-started/on-prem.md

```diff
@@ -107,8 +107,8 @@ export MEMBER_CLUSTER=YOUR-MEMBER-CLUSTER
 export MEMBER_CLUSTER_CONTEXT=YOUR-MEMBER-CLUSTER-CONTEXT
 
 # Run the script.
-chmod +x fleet/hack/membership/join.sh
-./fleet/hack/membership/join.sh
+chmod +x fleet/hack/hack/Azure/setup/joinMC.sh
```

Comment:
Maybe we should try to keep these consistent across the docs update the previous steps as well to match kind.md
```suggestion
chmod +x ./hack/Azure/setup/joinMC.sh
```